tasks/unit_tests/omnibus_tests.py

```diff
@@ -40,27 +41,45 @@ def _run_calls_to_string(mock_calls):
         'CI_PROJECT_DIR': '',
         'CI_PIPELINE_ID': '',
         'S3_OMNIBUS_GIT_CACHE_BUCKET': 'omnibus-cache',
-        'API_KEY_ORG2': 'api-key',
-        'AGENT_API_KEY_ORG2': 'agent-api-key',
-        'POD_NAMESPACE': os.environ.get('POD_NAMESPACE', 'gitlab-runner'),
     },
     clear=True,
 )
 class TestOmnibusCache(unittest.TestCase):
     def setUp(self):
         self.mock_ctx = MockContextRaising(run={})
 
+    @staticmethod
+    def _for_each_platform(calls_get_dd_api_key: bool = False):
+        def decorator(test_func):
+            @functools.wraps(test_func)
+            def wrapper(self, *args, **kwargs):
+                for platform, get_dd_api_key_env in {
+                    "darwin": {"AGENT_API_KEY_ORG2": "agent-api-key"},
+                    "linux": {"AGENT_API_KEY_ORG2": "agent-api-key", "POD_NAMESPACE": "pod-ns"},
+                    "win32": {"API_KEY_ORG2": "api-key"},
+                }.items():
```

Comment:
This is a point where I miss pytest 😬; interesting way to solve this regardless.

Reply:
I also miss `pytest`, but [nose](https://nose.readthedocs.io/en/latest/) even more.